src/GraphQL/Execution/ExecutionOptions.cs

```diff
@@ -29,5 +29,11 @@ public class ExecutionOptions
         public IFieldNameConverter FieldNameConverter { get; set; } = new CamelCaseFieldNameConverter();
 
         public bool ExposeExceptions { get; set; } = false;
+
+        //Note disabling will increase performance
+        public bool EnableLogging { get; set; } = true;
```

Comment:
I think this should be renamed to `EnableMetrics`.  Since logging is not the same things as the Metrics.

Reply:
Yeah i like this idea much better, this change was pushed.